junebug/workers.py

```diff
@@ -24,8 +33,24 @@ class MessageForwardingWorker(ApplicationWorker):
     CONFIG_CLASS = MessageForwardingConfig
 
     @inlineCallbacks
+    def setup_application(self):
+        self.redis = yield TxRedisManager.from_config(
+            self.config['redis_manager'])
+        message_redis = self.redis.sub_manager(
+            '%s:%s' % (
+                self.config['transport_name'],
+                self.config['inbound_message_prefix']))
```

Comment:
I think we usually include the `:` in the prefix itself. Would also be slightly DRYer, I guess, in case we changed the delimiter in one place but not the other(s).